dipy/workflows/reconst.py

```diff
@@ -21,6 +21,197 @@
 from dipy.reconst.shm import CsaOdfModel
 from dipy.workflows.workflow import Workflow
 
+from dipy.reconst import mapmri
+
+
+class ReconstMAPMRIFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'mapmri'
+
+    def run(self, data_file, data_bvecs, data_bvals, b0_threshold=0.0,
+            laplacian=True, positivity=True, bval_threshold=2000,
+            out_rtop='rtop.nii.gz', out_lapnorm='lapnorm.nii.gz',
+            out_msd='msd.nii.gz', out_qiv='qiv.nii.gz',
+            out_rtap='rtap.nii.gz',
+            out_rtpp='rtpp.nii.gz', out_ng='ng.nii.gz',
+            out_perng='perng.nii.gz',
+            out_parng='parng.nii.gz', small_delta=0.0129,
+            big_delta=0.0218, save_metrics=[],
+            out_dir='', laplacian_weighting=0.05, radial_order=6):
```

Comment:
Last wednesday, I learned that the parameter order has a crucial importance.  it should be : 
1. input ->  `data_file, data_bvecs, data_bvals`
2. optional parameter -> `b0_threshold=0.0, laplacian=True, positivity=True, bval_threshold=2000, small_delta=0.0129, big_delta=0.0218, save_metrics=[], laplacian_weighting=0.05, radial_order=6`
3. `out_dir`
4. All `out_*`

Can you do this change and move `small_delta=0.0129, big_delta=0.0218, save_metrics=[], out_dir='', laplacian_weighting=0.05, radial_order=6` in the right place ?

Can you do the same below too (in the documentation)

Thanks !

Reply:
Got it, I'll make the change shortly. Thanks for the note.